react/src/lib/components/DeckGLMap/layers/terrain/terrain-worker.ts

```diff
@@ -0,0 +1,1261 @@
+// Serve the worker as a local url.
+// See: https://deck.gl/docs/developer-guide/loading-data#loaders-and-web-workers
+// This is from the script terrain-worker.js but as a string. Some symbols are escaped.
```

Comment:
According to https://deck.gl/docs/developer-guide/loading-data#loaders-and-web-workers, there exist bundles in `node_modules/@loaders.gl/<module>/dist/<name>-loader.worker.js`. Should we instead of comitting the terrain worker script to our code base, instead include it in the bundle through `raw-loader` in `webpack` (or `assets/source` in `webpack 5`) - https://v4.webpack.js.org/loaders/raw-loader/?

Pros: Reduces the amount of code we "own", and perhaps makes it easier to stay in sync with any `@loaders.gl` bumps we do in `package.json` (since they then stays in sync).

Reply:
And also we are not ablde/allowed to use raw-loader in webpack 5. (Håvard would know more. I will remove the script from example data.